<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Gyro/Log Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_gyrometer">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>Gyrometer</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_Gyrometer"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_gyrometer_logmeter">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>Logmeter</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_logmeter"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QGroupBox" name="groupBox_42">
               <property name="title">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="0">
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>Gyro Reading</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_11">
                   <item row="0" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout_11">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="autoFillBackground">
                        <bool>true</bool>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                       </property>
                       <property name="text">
                        <string>Latitude:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_latitude">
                       <property name="styleSheet">
                        <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                  <zorder></zorder>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="title">
                   <string/>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_12">
                   <item row="0" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout_12">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="styleSheet">
                        <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                       </property>
                       <property name="text">
                        <string>Longitude:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_longitude">
                       <property name="styleSheet">
                        <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                  <zorder></zorder>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_43">
               <property name="title">
                <string>Log Meter Reading</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_14">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="styleSheet">
                     <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                    </property>
                    <property name="text">
                     <string>Speed:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_speed">
                    <property name="styleSheet">
                     <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Pitch/Roll Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_24">
            <property name="title">
             <string>Pitch</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_25">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_Pitch"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_25">
            <property name="title">
             <string>Roll</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_26">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_roll"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_26">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_31">
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBox_47">
               <property name="title">
                <string>Roll Reading</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_30">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_17">
                  <item>
                   <widget class="QLabel" name="label_33">
                    <property name="styleSheet">
                     <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                    </property>
                    <property name="text">
                     <string>Roll:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_Roll">
                    <property name="styleSheet">
                     <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_44">
               <property name="title">
                <string>Pitch Reading</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_27">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_16">
                  <item>
                   <widget class="QLabel" name="label_31">
                    <property name="styleSheet">
                     <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                    </property>
                    <property name="text">
                     <string>Pitch:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_Pitch">
                    <property name="styleSheet">
                     <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Wind Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QGroupBox" name="groupBox_30">
            <property name="title">
             <string>Wind Speed</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_33">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_WindSpeed"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_31">
            <property name="title">
             <string>Wind Direction</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_windDirection"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_32">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QGroupBox" name="groupBox_45">
                 <property name="title">
                  <string>Wind Reading</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_28">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_18">
                    <item>
                     <widget class="QLabel" name="label_35">
                      <property name="styleSheet">
                       <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                      </property>
                      <property name="text">
                       <string>Wind Speed:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_windSpeed">
                      <property name="styleSheet">
                       <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_46">
               <property name="title">
                <string>Wind Reading</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_29">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_19">
                  <item>
                   <widget class="QLabel" name="label_37">
                    <property name="styleSheet">
                     <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                    </property>
                    <property name="text">
                     <string>Wind Direction:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_WindDirection">
                    <property name="styleSheet">
                     <string notr="true">font-size:25px;
color:black;
font-weight:bold;</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Destination</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_23">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_19">
            <item row="0" column="0">
             <widget class="QLabel" name="label_15">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Bearing:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_Destination_Bearing">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>WayPoint</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string/>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_23">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Bearing:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_WayPoint_Bearing">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_25">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Range  :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_wayPoint_distance">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string/>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_19">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Lat  :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_waypoint_Lat">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_21">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Long:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_waypoint_Long">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
        <zorder>groupBox_5</zorder>
        <zorder>groupBox_6</zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_22">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_18">
            <item row="0" column="0">
             <widget class="QLabel" name="label_16">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Current Time:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_Date_Time">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_20">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_16">
            <item row="0" column="1">
             <widget class="QLabel" name="label_Altitude">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Alt. :</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_21">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="0" column="1">
             <widget class="QLabel" name="label_rudder">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Rudder:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_19">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>Water temp:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_waterTemp">
              <property name="styleSheet">
               <string notr="true">font-size:12px;
color:black;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1224</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
